app/policies/medical_copays_policy.rb

```diff
@@ -8,12 +8,12 @@
   # @return [Boolean]
   #
   def access?
-    accessible = Flipper.enabled?('show_medical_copays', user) &&
-                 user.edipi.present? &&
-                 user.icn.present?
-
-    increment_statsd(accessible)
-
+    accessible = user.edipi.present? && user.icn.present?
+    if accessible
+      StatsD.increment('api.mcp.policy.success')
+    else
+      StatsD.increment('api.mcp.policy.failure')
+    end
```

Comment:
**Suggestion (non-blocking):** If we want to get even more concise here, we could try this:

```ruby
StatsD.increment("api.mcp.policy.#{accessible ? `success` : 'failure'}")
```

Reply:
@maxx1128 have a look now